src/ui/popup/manage-settings-page/style.less

```diff
@@ -0,0 +1,31 @@
+@import "../../theme";
+
+.manage-settings-page {
+
+    &__button--active {
```

Comment:
Is there a need in these rules? I think you can leave `.manage-settings-page {}` empty at the moment (it will be needed later).